src/trading/Order.sol

```diff
@@ -102,20 +102,20 @@ library Order {
     }
 
     function saveOrder(Order.Data _orderData, uint256 _tradeGroupId) internal returns (bytes32) {
-        return _orderData.orders.saveOrder(_orderData.tradeType, _orderData.market, _orderData.fxpAmount, _orderData.fxpPrice, _orderData.maker, _orderData.outcome, _orderData.fxpMoneyEscrowed, _orderData.fxpSharesEscrowed, _orderData.betterOrderId, _orderData.worseOrderId, _tradeGroupId);
+        return _orderData.orders.saveOrder(_orderData.tradeType, _orderData.market, _orderData.amount, _orderData.price, _orderData.maker, _orderData.outcome, _orderData.moneyEscrowed, _orderData.sharesEscrowed, _orderData.betterOrderId, _orderData.worseOrderId, _tradeGroupId);
     }
 
     //
     // Private functions
     //
 
     function escrowFundsForBid(Order.Data _orderData) private returns (bool) {
-        uint256 _orderValueInAttotokens = SafeMathUint256.fxpMul(_orderData.fxpAmount, SafeMathUint256.sub(_orderData.market.getCompleteSetCostInAttotokens(), uint256(_orderData.fxpPrice)), 1 ether);
+        uint256 _orderValueInAttotokens = _orderData.amount.mul(SafeMathUint256.sub(_orderData.market.getMarketDenominator(), _orderData.price));
```

Comment:
Recommend using the extension method syntax here for the subtraction, and also recommend chaining over nesting:
```
_orderData.market.getMarketDenominator().sub(_orderData.price).mul(_orderData.amount)
```
Alternatively, without chaining (but still with extension calling convention):
```
_orderData.amount.mul(_orderData.market.getMarketDenominator().sub(_orderData.price))
```